SubscribeIT/ngDesk
# Teams drop down

Closes #233 

Test Case 1
Name:-Teams Drop down in create layout
1. Login to ngdesk.
2. Goto -> tickets -> new.
3. Click on teams field.
4. Remove global.
5. Observe global is enabled in drop down.
6. Fill details and hit save.
7. Observe getting saved.
8. Observe no error.

Test Case 2
Name:-Teams Drop down in edit layout
1. Login to ngdesk.
2. Goto -> tickets -> click on ticket created above.
3. Click on teams field.
4. Remove the selcted team.
5. Observe the selected team is enabled in drop down.
6. Fill details and hit save.
7. Observe getting updated.
8. Observe no error.

Screenshots :- 

![Screenshot from 2021-10-07 16-59-34](https://user-images.githubusercontent.com/89504119/136375796-72571a5f-c69b-4a22-8936-7b5a77cc8b03.png)
![Screenshot from 2021-10-07 16-59-52](https://user-images.githubusercontent.com/89504119/136375798-0a5df24a-2218-4333-83b5-87800dfa1e1b.png)
![Screenshot from 2021-10-07 17-00-01](https://user-images.githubusercontent.com/89504119/136375801-3a4c8a5e-5d7d-4506-be13-d5c12270af8e.png)
